index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header id="header">
+        <img id="header-img" src="images/product-landing-page-logo.png">
```

Comment:
- [ ] Please try to be more professional, clear and polite in your comments and add solid reasons. Kindly follow the below comment to see how to perform code reviews with effective communication through text.
   - Kindly add the images or video to your repository instead of adding the 3rd party links like youTube links because in the near future, if the owner of the video deletes it from youTube then still anybody can see the video or image that you are using without any inconvenience.  

## Your comment

>  as much as you can try to make you images and video source from you own repository the video you use is from youtube some one channel so if he delete it you will lose so try to put the file in you own folder and path to that one